routing/async_router.cpp

```diff
@@ -227,6 +230,15 @@ void AsyncRouter::LogCode(IRouter::ResultCode code, double const elapsedSec)
     case IRouter::IntermediatePointNotFound:
       LOG(LWARNING, ("Can't find intermediate point node"));
       break;
+    case IRouter::TransitRouteNotFoundNoNetwork:
+      LOG(LWARNING, ("Transit route not found because no transit network in route point mwm"));
+      break;
+    case IRouter::TransitRouteNotFoundTooLongPedestrian:
+      LOG(LWARNING, ("Transit route not found because pedestrian way is too long"));
+      break;
+    case IRouter::RouteNotFoundRedressRouteError:
+      LOG(LWARNING, ("Route not found because a redress route error"));
```

Comment:
s/because/because of/

Reply:
Done

Reply:
https://github.com/mapsme/omim/pull/8000